config/initializers/ldap_authenticatable.rb

```diff
@@ -3,15 +3,24 @@ module Strategies
     class LdapAuthenticatable < Authenticatable
 
       def authenticate!
-        if valid_credentials?("#{username}@chapman.edu", password)
+        if valid_credentials?("#{username}@chapman.edu", password)          
```

Comment:
Remove extra spaces at the end of this line, and also the extra newline on line 23 (these are just best practices). We started using the [Rubocop gem](https://github.com/bbatsov/rubocop) in all of our rails apps moving forward, which is a Ruby code linter, and it would've caught these for sure. The repos that have it so far are cascade-assets, newsroom and imposter.